backend/src-common/src/main/java/org/eclipse/sw360/datahandler/entitlement/ReleaseModerator.java

```diff
@@ -10,6 +10,7 @@
 
 import org.eclipse.sw360.datahandler.common.Moderator;
 import org.eclipse.sw360.datahandler.common.SW360Utils;
+import org.eclipse.sw360.datahandler.thrift.ReleaseRelationship;
```

Comment:
No. Used at https://github.com/sw360/sw360portal/pull/395/files/49c7da2a83d3922cee6c0d8765bb9ebdf4c0471d#diff-5ae0339c1ee942a32403fc4eaef8aadaR103
Used to be imported by line https://github.com/sw360/sw360portal/pull/395/files/49c7da2a83d3922cee6c0d8765bb9ebdf4c0471d#diff-5ae0339c1ee942a32403fc4eaef8aadaR16